csharp/src/Google.Protobuf/WellKnownTypes/TimestampPartial.cs

```diff
@@ -222,6 +222,109 @@ internal static string ToJson(long seconds, int nanoseconds, bool diagnosticOnly
             }
         }
 
+        /// <summary>
+        /// Given another timestamp, returns 0 if the timestamps are equivalent, -1 if this timestamp precedes the other, and 1 otherwise
+        /// </summary>
+        /// <remarks>
+        /// The result of comparing non-normalized timestamps is not specified and may give unexpected results
```

Comment:
"Make sure the timestamps are normalized.  Comparing non-normalized timestamps is not specified and may give unexpected results."